src/app/backend/replicasetdetail.go

```diff
@@ -73,13 +81,33 @@ func GetReplicaSetDetail(client *client.Client, namespace string, name string) (
 		return nil, err
 	}
 
+	services, err := client.Services(namespace).List(labels.Everything(), fields.Everything())
+
+	if err != nil {
+		return nil, err
+	}
+
+	matchingServices := getMatchingServices(services.Items, replicaSet)
```

Comment:
This part is taken and adjusted from replica set list in order to add details about internal/external endpoints. Should it be changed?